internal/fullnode/pvc_builder_test.go

```diff
@@ -148,30 +149,81 @@ func TestBuildPVCs(t *testing.T) {
 		}
 	})
 
-	t.Run("pvc auto scale", func(t *testing.T) {
-		for _, tt := range []struct {
-			SpecQuant, AutoScaleQuant, WantQuant string
-		}{
-			{"100G", "99G", "100G"},
-			{"100G", "101G", "101G"},
-		} {
-			crd := defaultCRD()
-			crd.Spec.Replicas = 1
-			crd.Spec.VolumeClaimTemplate = cosmosv1.PersistentVolumeClaimSpec{
-				Resources: corev1.ResourceRequirements{
-					Requests: corev1.ResourceList{corev1.ResourceStorage: resource.MustParse(tt.SpecQuant)},
-				},
-			}
-			crd.Status.SelfHealing.PVCAutoScale = &cosmosv1.PVCAutoScaleStatus{
-				RequestedSize: resource.MustParse(tt.AutoScaleQuant),
+	t.Run("pvc auto scale with padding", func(t *testing.T) {
```

Comment:
I kept the tests the same, but will cleanup move reusable bits outs

Reply:
This test duplication is fine to me (even preferred) because it's isolated to a single test file.